app/controllers/users_controller.rb

```diff
@@ -1978,9 +1982,17 @@ def user_menu_messages
     if messages_list
       serialized_messages =
         serialize_data(messages_list, TopicListSerializer, scope: guardian, root: false)[:topics]
+      serialized_users =
```

Comment:
This is again supporting the "hack" that we use to populate posters with actual user data... I would LOVE to change `TopicListSerializer` to support this cleanly, but that feels like the most active/important serializer and I'm hesitant to do that.